Common subdirectories: ../../default/html/.svn and ./source/html/.svn
Common subdirectories: ../../default/html/images and ./source/html/images
diff -bNau ../../default/html/imp_exp_text_details.html ./source/html/imp_exp_text_details.html
--- ../../default/html/imp_exp_text_details.html	2010-09-05 00:12:11.000000000 +0400
+++ ./source/html/imp_exp_text_details.html	2010-11-04 13:55:47.000000000 +0300
@@ -58,23 +58,23 @@
 Title Row for full data export currently contains&nbsp;the
 following fields by default, separated by a tab.</span></p>
 <ul>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Group/Title</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Username</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Password</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">URL</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">AutoType</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Created Time</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Password Modified Time</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Last Access Time</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Password Expiry Date</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Password Expiry Interval</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Record Modified Time</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Password Policy</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">History</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Run Command</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">DCA</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">e-mail</span></li>
-  <li><span style="font-size: 11pt;" lang="EN-GB">Notes</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Group/Title_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Username_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Password_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_URL_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_AutoType_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Created Time_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Password Modified Time_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Last Access Time_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Password Expiry Date_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Password Expiry Interval_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Record Modified Time_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Password Policy_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_History_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Run Command_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_DCA_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_e-mail_</span></li>
+  <li><span style="font-size: 11pt;" lang="EN-GB">_Notes_</span></li>
 </ul>
 <p class="CitiChangeSummarybullet" style="margin-left: 0in; text-align: justify;"><span style="font-size: 11pt;" lang="EN-GB"><span style=""></span>When
 exporting data, you
@@ -119,67 +119,67 @@
 blank, empty or contain the delimiter character.</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Username</td>
+      <td style="padding: 0in 5.4pt;">_Username_</td>
       <td style="padding: 0in 5.4pt;"> </td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Password</td>
+      <td style="padding: 0in 5.4pt;">_Password_</td>
       <td style="padding: 0in 5.4pt;">Cannot be blank or empty.</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">URL</td>
+      <td style="padding: 0in 5.4pt;">_URL_</td>
       <td style="padding: 0in 5.4pt;"> </td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Autotype</td>
+      <td style="padding: 0in 5.4pt;">_Autotype_</td>
       <td style="padding: 0in 5.4pt;"> </td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Created Time</td>
+      <td style="padding: 0in 5.4pt;">_Created Time_</td>
       <td style="padding: 0in 5.4pt;">See note <a href="#1"><sup>1</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Password Modified Time</td>
+      <td style="padding: 0in 5.4pt;">_Password Modified Time_</td>
       <td style="padding: 0in 5.4pt;">See note <a href="#1"><sup>1</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Last Access Time</td>
+      <td style="padding: 0in 5.4pt;">_Last Access Time_</td>
       <td style="padding: 0in 5.4pt;">See note <a href="#1"><sup>1</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Password Expiry Date</td>
+      <td style="padding: 0in 5.4pt;">_Password Expiry Date_</td>
       <td style="padding: 0in 5.4pt;">See note <a href="#1"><sup>1</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Password Expiry Interval</td>
+      <td style="padding: 0in 5.4pt;">_Password Expiry Interval_</td>
       <td style="padding: 0in 5.4pt;">See note <a href="#1"><sup>1</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Record Modified Time</td>
+      <td style="padding: 0in 5.4pt;">_Record Modified Time_</td>
       <td style="padding: 0in 5.4pt;">See note <a href="#1"><sup>1</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Password Policy</td>
+      <td style="padding: 0in 5.4pt;">_Password Policy_</td>
       <td style="padding: 0in 5.4pt;">See note <a href="#3"><sup>3</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">History</td>
+      <td style="padding: 0in 5.4pt;">_History_</td>
       <td style="padding: 0in 5.4pt;">See note <a href="#2"><sup>2</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Run command</td>
+      <td style="padding: 0in 5.4pt;">_Run command_</td>
       <td style="padding: 0in 5.4pt;"></td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">DCA</td>
+      <td style="padding: 0in 5.4pt;">_DCA_</td>
       <td style="padding: 0in 5.4pt;"> = &lsquo;Double Click Action&rsquo;. See note <a href="#4"><sup>4</sup></a> below</td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">e-mail</td>
+      <td style="padding: 0in 5.4pt;">_e-mail_</td>
       <td style="padding: 0in 5.4pt;"></td>
     </tr>
     <tr>
-      <td style="padding: 0in 5.4pt;">Notes</td>
+      <td style="padding: 0in 5.4pt;">_Notes_</td>
       <td style="padding: 0in 5.4pt;">
       <ul>
         <li>The Notes field should be enclosed by double quotes ("like this").</li>
diff -bNau ../../default/html/manage_menu.html ./source/html/manage_menu.html
--- ../../default/html/manage_menu.html	2010-09-05 00:12:11.000000000 +0400
+++ ./source/html/manage_menu.html	2010-11-04 12:26:50.000000000 +0300
@@ -19,7 +19,7 @@
 options and settings are controlled. </p>
 
 <div class="instr">
-<h3>Change Safe Combination</h3>
+<h3>Change Safe Combination...</h3>
 
 <p> Selecting this option brings up the Change Safe Password
 dialog.<br>
@@ -35,7 +35,7 @@
 Up and Restoring Databases</a> for more information.</i></small>
 </p>
 
-<h3>Restore from Backup</h3>
+<h3>Restore from Backup...</h3>
 
 <p>Restore a backed up password database file.<br>
 
@@ -73,7 +73,7 @@
 <small>Many of the setting are stored as <span style="font-style: italic;">preferences</span>, as
 described <a href="preferences.html">here</a>.</small>
 </div>
-<h3>Validate</h3>
+<h3>Validate...</h3>
 <p>After selecting a database, this database will be checked  for being completely compatible to  <b>PassowrdSafe</b>.<br>
 
 <small>Note: This menu item will only be enabled if there is no&nbsp;database currently opened.</small>
diff -bNau ../../default/html/menu_toolbar.html ./source/html/menu_toolbar.html
--- ../../default/html/menu_toolbar.html	2010-09-05 00:12:11.000000000 +0400
+++ ./source/html/menu_toolbar.html	2010-11-04 12:27:28.000000000 +0300
@@ -31,8 +31,8 @@
 <table align="center" cellspacing="5">
   <tbody>
     <tr bgcolor="#003333">
-      <td align="center" width="10%"><b><font color="white">Classic</font></b></td>
-      <td align="center" width="10%"><b><font color="white">New</font></b></td>
+      <td align="center" width="10%"><b><font color="white">Classic style</font></b></td>
+      <td align="center" width="10%"><b><font color="white">New style</font></b></td>
       <td align="center" width="10%"><b><font color="white">Default</font></b></td>
       <td><b><font color="white">Description</font></b></td>
     </tr>
diff -bNau ../../default/html/password_history.html ./source/html/password_history.html
--- ../../default/html/password_history.html	2010-09-05 00:12:11.000000000 +0400
+++ ./source/html/password_history.html	2010-11-04 22:16:52.000000000 +0300
@@ -62,8 +62,7 @@
 
 <img alt="Password History" src="images/edit_view_entry_add.jpg">
 <p>This dialog box displays the last few previous passwords associated
-with a given entry. To display this, click on the "Password History"
-button in the Edit/View Entry dialog box. You can do the following
+with a given entry. You can do the following
 operations here:</p>
 
 
@@ -87,7 +86,7 @@
 
 
 Clicking on a row in the list of recent passwords will copy the password displayed in that row&nbsp;to the clipboard.
-<h3>Clear History</h3>
+<h3>Clear</h3>
 
 
 
diff -bNau ../../default/html/style.css ./source/html/style.css
--- ../../default/html/style.css	2010-09-05 00:12:11.000000000 +0400
+++ ./source/html/style.css	2011-02-09 15:40:19.000000000 +0300
@@ -26,19 +26,23 @@
 	{
 	margin-top: .75em;
 	margin-bottom: 1.25em;	
+	text-align:justify;
 	}	
 
 li	{
 	margin-left: .15in;
+	text-align:justify;
 	}	 
 
 ol li
 	{
 	margin-left: .10in;
+	text-align:justify;
 	}
 
 ul	{
 	list-style: disc;
+	text-align:justify;
 	}
 
 ul.onestep
@@ -73,3 +77,9 @@
 	vertical-align: top;
 	padding: 0;
 	}
+
+p	{
+        text-align:justify;
+	text-indent: 3ex;
+	margin:0.25em;
+	}
diff -bNau ../../default/html/wzselectdb.html ./source/html/wzselectdb.html
--- ../../default/html/wzselectdb.html	2011-02-07 18:45:12.000000000 +0300
+++ ./source/html/wzselectdb.html	2011-02-09 19:50:30.000000000 +0300
@@ -39,13 +39,19 @@
 based on the contents of the Group, Title, User Name, URL or Notes fields and/or which
 fields to Compare, Merge, Synchronize or Export.</p>
 
-<p>Note 1: When exporting a single entry via the Edit Menu or Context Menu (by right
+<hr>
+<small><b>Notes:</b><br></small>
+<ul>
+<li><small>Note 1: When exporting a single entry via the Edit Menu or Context Menu (by right
 clicking on the entry), the option to restrict the export based on the contents of
-the Group, Title, User Name, URL or Notes fields is obviously not available.</p>
-
-<p>Note 2: When Merging a database into the currently open database, you will not be able
+the Group, Title, User Name, URL or Notes fields is obviously not available.</small>
+</li>
+<li><small>Note 2: When Merging a database into the currently open database, you will not be able
 to restrict which fields of the entries being merged are copied.  If an entry passes your
-selection criteria, then the complete entry will be merged into the current database.</p>
+selection criteria, then the complete entry will be merged into the current database.</small>
+</li>
+</ul>
+<hr>
 <br>
 <br>
 <br>
